game/kernel/jak1/kscheme.cpp

```diff
@@ -1712,10 +1725,9 @@ s32 InitHeapAndSymbol() {
     // check the kernel version!
     auto kernel_version = intern_from_c("*kernel-version*")->value;
     if (!kernel_version || ((kernel_version >> 0x13) != KERNEL_VERSION_MAJOR)) {
-      MsgErr("\n");
-      MsgErr(
-          "dkernel: compiled C kernel version is %d.%d but the goal kernel is %d.%d\n\tfrom the "
-          "goal> prompt (:mch) then mkee your kernel in linux.\n",
+      lg::error(
+          "Kernel version mismatch! Compiled C kernel version is {}.{} but"
+          "the goal kernel is {}.{}",
```

Comment:
Thanks. For a historical note - we used `MsgErr` and this weird error message because that's what was in the original game. At the time this was written, I was more interested in matching the original game, but we've now shifted towards modernizing the C++ code, and using `lg::error` is a lot nicer.